documentation/asciidoc/topics/proc_creating_caches.adoc

```diff
@@ -0,0 +1,41 @@
+[id='creating-caches_{context}']
+= Creating caches with the Cache CR
+
+[role="_abstract"]
+Complete the following steps to create caches on {datagridservice} clusters using valid configuration in XML or YAML format.
+
+.Procedure
+
+. Create a `Cache` CR with a unique value in the `metadata.name` field.
+. Specify the target {brandname} cluster with the `spec.clusterName` field.
+. Name your cache with the `spec.name` field.
++
+[NOTE]
+====
+The `name` attribute in the cache configuration is ignored.
+Only the `spec.name` field applies to the resulting cache.
```

Comment:
This isn't strictly true. The cache is named using `spec.name`, but if this is omitted `meta.name` is used instead. Is this covered elsewhere?

Reply:
@ryanemerson If you omit `spec.name` then the cache gets the same name as the Cache CR supplied with `metadata.name`?

Does the name in the cache configuration ever take effect when you create via Cache CR? I'll try this out.

Reply:
>If you omit spec.name then the cache gets the same name as the Cache CR supplied with metadata.name?

Yep

> Does the name in the cache configuration ever take effect when you create via Cache CR?

It shouldn't!

Reply:
I've updated with this. Thanks for pointing that out about the `metadata.name` field.